src/pages/Create.tsx

```diff
@@ -60,10 +62,21 @@ import { enableWebln } from "../utils/webln";
 const Create = () => {
     let receiveAmountRef: HTMLInputElement, sendAmountRef: HTMLInputElement;
 
+    // if btc and amount > 9, switch to sat
+    // user failed to notice the non satoshi denomination
+    const changeDenomination = (amount: number) => {
+        if (denomination() === denominations.btc && amount > 9) {
+            setDenomination(denominations.sat);
+        } else if (denomination() === denominations.sat && amount < 1) {
```

Comment:
This has the side effect of switching to BTC denom when the amount is cleared. Questionable if we want that

Reply:
We don't. No switching on clearing.

Reply:
is it better now?  :)